src/Agoda.Analyzers.CodeFixes/AgodaCustom/AG0020FixProvider.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Agoda.Analyzers.AgodaCustom;
+using Agoda.Analyzers.Helpers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Agoda.Analyzers.CodeFixes.AgodaCustom
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(AG0020FixProvider))]
+    [Shared]
+    public class AG0020FixProvider : CodeFixProvider
+    {
+        // this isn't very nice, we need to move IDs to a shared const class
```

Comment:
Could you just use `AG0020AvoidReturningNullEnumerables.DiagnosticId`?

Reply:
fixed